spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/annotation/ContinueSpan.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
It could be of value since sleuth has an opinion inherited by spring boot etc. spring data may be compatible with those opinions but others may not be.

regardless, we can wait until more request an "api jar" as that can be spun out at any time. There's also a possibility to allow any annotation!